libcudacxx/test/libcudacxx/cuda/ptx/generated/ld.h

```diff
@@ -101,7 +101,7 @@ __global__ void test_ld(void** fn_ptr)
                (
                    // ld.global.v4.b64 dest, [addr];
                    * fn_ptr++ = reinterpret_cast<void*>(
-                     static_cast<longlong4 (*)(cuda::ptx::space_global_t, const longlong4*)>(cuda::ptx::ld));));
+                     static_cast<longlong4_32a (*)(cuda::ptx::space_global_t, const longlong4_32a*)>(cuda::ptx::ld));));
```

Comment:
This would break with earlier CTKs pre 13.0, which do not know this type. We either need to define an alias or suppress the warning

Reply:
The new vector types are defined for older CTKs in `ptx_helper_functions.h`, which is a problem in itself. See my comment below.